README.md

```diff
@@ -18,6 +18,14 @@ Many steps of the build process require considerable privilege.  It is
 highly recommend to build this project as root.  You may find some success
 with `sudo` if you know what you're doing.
 
+## Additional Software
+
+Podman is required to use the tool.  On Oracle Linux systems it can be installed
+via `dnf install podman`.  When building qcow2 images, the `qemu-img` utility is
```

Comment:
minor - Per https://docs.oracle.com/en/operating-systems/oracle-linux/podman/podman-InstallingPodmanandRelatedUtilities.html#podman-install, you need to do the following before actually installing podman package:
Oracle Linux 8
```sudo dnf module install container-tools:ol8```
Oracle Linux 9
```sudo dnf install container-tools```

Reply:
I ended up linking to the install guide directly.